Dockerfile

```diff
@@ -25,17 +25,10 @@ RUN sudo apt-get update \
     cmake \
   && curl -sSL https://rvm.io/mpapis.asc | gpg --import -
 
-# install RVM's ordered dependencies and repos
-RUN sudo apt-get install software-properties-common \
-  && sudo apt-add-repository -y ppa:rael-gc/rvm \
-  && sudo apt-get update \
-  && sudo apt-get install -y libfile-fcntllock-perl
+USER dev.user
```

Comment:
Does the user context change between line 14 and line 28, that necessitates resetting `USER dev.user`